docs/tutorials/01-intro.md

```diff
@@ -20,6 +24,14 @@ Then install buildah by running:
 
     # dnf -y install buildah
 
+## Rootless User Configuration
+
+If you plan to run Buildah as a user without root privileges, i.e. a "rootless user", the administrator of the system will have 
```

Comment:
This should say 
the administrator of the system might have 

In general on a Fedora or Ubuntu system the /etc/subuid and /etc/subgid files should be setup correctly and no admin action required.

Reply:
fixed

Reply:
Also added a clarification that Podman's config/setup requirements are the same as Buildah's.